symfony/test/functional/cypress/fixtures/dummy.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Maybe rename files as string.json

Reply:
Not to be pedantic but this file has both string and integers. but i renamed it to chars just to be polymorphic and valid is all cases